docs/understand/hipgraph.rst

```diff
@@ -0,0 +1,91 @@
+.. meta::
+    :description: This chapter provides an overview over the usage of HIP graph.
+    :keywords: ROCm, HIP, graph, stream
+
+.. understand_HIP_graph:
+
+********************************************************************************
+HIP graph
+********************************************************************************
+
+HIP graphs are an alternative way of executing work on a GPU. It can provide
```

Comment:
I would eliminate this first paragraph, leave the following Beta note in place, and start this _Understanding HIP Graphs_ topic with the _Graph format_ text (but without the header).

Reply:
Done